### ISSUE TITLE
teamspeak_client has no .desktop file

### ISSUE THREAD

**Issue by sjau** [NixOS/nixpkgs#8858]
Hi there

I installed teamspeak_client but no menu entry for KDE4 was added. I assume it's lacking a .desktop file.

I did create one manually with k-menu.

**Comment by nathanielbaxter** [comment]
Hey,
You're correct that the teamspeak client doesn't make a .desktop file. I'm the maintainer for teamspeak-client, but I don't use a file manager/menu to run apps so I don't notice these things. :) I can add an item, although there don't seem to be any good icons for it. Did you find a decent icon for it?

Btw, if you have an issue/PR related to a specific package then it's best to mention them in your issue so that they get notified about it. You can find the maintainer/s from the teamspeak default.nix definition metadata and the lib/maintainers.nix file in this repo.

**Comment by sjau** [comment]
If you download the the package from their site and extract it, there's a ts3icon.png available.

Not sure what you mean by your secend paragraph though.

**Comment by nathanielbaxter** [comment]
Which package are you referring to? Neither the Linux "Client amd64 3.0.16" nor "Client x86 3.0.16" packages (which are used by the nixpkgs package) contain a ts3icon.png file.

***Btw, if you have an issue/PR related to a specific package then it's best to mention the package maintainer in your issue so that they receive a notification. You can find maintainer/s by searching for the package in the git repo. (also lib/maintainers.nix has a map from nixpkgs maintainer name to email which can help if github account name is different from name in nixpkgs)

**Comment by sjau** [comment]
Oh... right.... hmmm, then I wonder where I got it from. Might have been copied over from Windows.

I still don't understand why to contact the maintainer directly? I thought the whole point of github was (among having git) that maintainers of the packages would also get notified when an issue is opened.

**Comment by nathanielbaxter** [comment]
Well, I personally have email notifications for nixpkgs turned off (unless someone mentions me), as there are too many pull requests/issues opened that aren't relevant to packages that I maintain, I suspect this applies to most nixpkgs package maintainers as there are a lot of them. The difference from most github projects being that we have lots of maintainers for packages within the project. You don't *have* to do it, but it makes it much more likely that you'll get a response (I only saw this because a friend was reading through issues and pointing me to it).

Apparently the client pluginsdk has a decent icon. Added in #8972.
